frontend/src/metabase/lib/errors/messages.ts

```diff
@@ -26,3 +26,7 @@ export function getResponseErrorMessage(error: unknown): string | undefined {
 
   return undefined;
 }
+
+export function isResourceNotFoundError(error: any) {
+  return error?.status === 404;
+}
```

Comment:
This is fine as I couldn't find an error type with `{status?: number}` in the codebase

Reply:
🙏 I don't wanna fix any type right now, I don't remember why I went with `any` but it seems using `GenericErrorResponse | null` which is the type where the argument is passed doesn't pass TypeScript here. So a deliberate `any` is a compromise.